lib/rspec_api_documentation/writers/formatter.rb

```diff
@@ -3,9 +3,9 @@ module Writers
     module Formatter
 
       def self.to_json(object)
-        JSON.pretty_generate(object.as_json)
+        JSON.generate(object.as_json)
```

Comment:
Additional data: This issue (the java.lang.NegativeArraySizeException problem) was solved by upgrading Rails from 4.0.13 to 4.1.8 (and probably more importantly ActiveSupport from 4.0.13 to 4.1.8, as it's ActiveSupport implicated in the json tickets). It may just be a coincidence that that fixes it, but that works for us!